src/App.js

```diff
@@ -80,7 +80,7 @@ const App = () => {
        * Uses React history directly instead of browser history to avoid template realod.
        * only redirect after first application mount
        */
```

Comment:
Does this explanation comment need updating since we aren't using `!ignoreRedirect` anymore?